katharsis-client/src/main/java/io/katharsis/client/internal/RelationshipRepositoryStubImpl.java

```diff
@@ -0,0 +1,126 @@
+package io.katharsis.client.internal;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.squareup.okhttp.HttpUrl;
+import com.squareup.okhttp.Request;
+import com.squareup.okhttp.Request.Builder;
+import com.squareup.okhttp.RequestBody;
+
+import io.katharsis.client.KatharsisClient;
+import io.katharsis.client.RelationshipRepositoryStub;
+import io.katharsis.queryParams.QueryParams;
+import io.katharsis.request.path.JsonPath;
+import io.katharsis.request.path.ResourcePath;
+import io.katharsis.resource.field.ResourceField;
+import io.katharsis.resource.information.ResourceInformation;
+import io.katharsis.resource.registry.RegistryEntry;
+import io.katharsis.response.BaseResponseContext;
+import io.katharsis.response.CollectionResponseContext;
+import io.katharsis.response.JsonApiResponse;
+import io.katharsis.response.LinkageContainer;
+import io.katharsis.response.ResourceResponseContext;
+import io.katharsis.utils.PropertyUtils;
+
+public class RelationshipRepositoryStubImpl<T, T_ID extends Serializable, D, D_ID extends Serializable>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this generic name to match the regular expression '^[A-Z][0-9]?$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00119)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this generic name to match the regular expression '^[A-Z][0-9]?$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00119)